### PR TITLE
Mccalluc/czems tweaks

### LINKED ISSUE #857: Create cems schema
from https://docs.google.com/spreadsheets/d/1CSlYduFtki-j8p1iqglZsYPlELtyHwwrOdOSEgzIBYw/edit?usp=sharing

### PR DESCRIPTION
**Pull request by mccalluc**
Extend #859 
Fix #857

Please resolve the TODOs.

### CODE REVIEW

**Review by cebriggs7135 — CHANGES_REQUESTED** [review]
I think we need to have a new assay_category called mass spectrometry and GC-MS and CE-MS will be assay_types of mass_spectrometry.
I'll likely have more edits in the definitions.

File: docs/czems/czems-metadata.tsv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+version	description	donor_id	tissue_id	execution_datetime	protocols_io_doi	operator	operator_email	pi	pi_email	assay_category	assay_type	analyte_class	is_targeted	acquisition_instrument_vendor	acquisition_instrument_model	ms_source	polarity	mz_range_low_value	mz_range_high_value	mass_resolving_power	mz_resolving_power	ion_mobility	data_collection_mode	ms_scan_mode	labeling	section_prep_protocols_io_doi	ce_interface	ce_capillary_coating	ce_background_electrolyte	ce_instrument_vendor	ce_instrument_model	ce_electroosmotic_flow	spatial_type	spatial_sampling_type	spatial_target	resolution_x_value	resolution_x_unit	resolution_y_value	resolution_y_unit	processing_search	processing_protocols_io_doi	overall_protocols_io_doi	contributors_path	data_path
+1										mass_spectrometry_imaging	CZE-MS	protein / metabolites / lipids / peptides / phosphopeptides / glycans				ESI / MALDI / DESI	negative ion mode / positive ion mode					TIMS / TWIMS / FAIMS / DTIMS / SLIMS		MS / MS/MS / MS3			sheath-flow / sheathless					suppressed / normal / reversed	LCM / LESA / nanoPOTS / microLESA	profiling / imaging			nm / um		nm / um
```
[the remark this answers is not in the document]
Comment (mccalluc) [reply docs/czems/czems-metadata.tsv:2]: Remember to update src, not TSV.

**Comment by mccalluc** [comment]
@cebriggs7135 -- Lots of TODOs in the schema need to be addressed.

**Comment by mccalluc** [comment]
@cebriggs7135 -- As discussed, I will merge this to the parent branch, and you can confirm that the over-all list of fields is correct, and and the descriptions can be handled later.

**Pull request merged**